unit-tests/live/options/test-timestamp-domain.py

```diff
@@ -0,0 +1,117 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+
+import time
+import pyrealsense2 as rs
+
+from rspy import test
+
+
+def get_connected_device():
+    """
+    Get first connected device
+    :return: connected device
+    """
+    dev = None
+    try:
+        context = rs.context()
+        dev = context.devices[0]
+    except Exception as ex:
+        print('Exception: Failed to find connected device. ', str(ex))
+    return dev
+
+
+def close_resources(sensor):
+    """
+    Stop and Close sensor.
+    :param: sensor of device
+    """
+    if len(sensor.get_active_streams()) > 0:
+        sensor.stop()
+        sensor.close()
+
+
+def start_sensor_test(sensor, global_time_enabled: int):
+    """
+    Perform depth sensor test according given global time
+    :global_time_enabled int: 1 - time is enabled, 0 - time disabled
+    """
+
+    if global_time_enabled not in [0, 1]:
+        raise ValueError(f'Invalid parameter in start depth test: {global_time_enabled}. Only 1 or 0 are valid.')
+
+    try:
+        sensor.set_option(rs.option.global_time_enabled, global_time_enabled)
+        time.sleep(2)
+
+        frame = frame_queue.wait_for_frame()
+
+        if global_time_enabled == 0 and sensor.get_option(rs.option.global_time_enabled) == 0:
+            test.check_equal(frame.get_frame_timestamp_domain(), rs.timestamp_domain.hardware_clock)
+
+        elif global_time_enabled == 1 and sensor.get_option(rs.option.global_time_enabled) == 1:
+            test.check_equal(frame.get_frame_timestamp_domain(),
+                             rs.timestamp_domain.global_time)
+
+    except Exception as exc:
+        print(str(exc))
+
+
+depth_sensor = None
+color_sensor = None
+
+try:
+    device = get_connected_device()
+    depth_sensor = device.first_depth_sensor()
+    color_sensor = device.first_color_sensor()
+
+    depth_profile = next(p for p in depth_sensor.profiles if p.fps() == 30
```

Comment:
I know this is done the same in most of our tests but it's limit us to work if all devices support this settings.
For this test we can use the default format as well..
I don't see a reason to enforce this specific resolution and format right?

Reply:
Changed to:
```python
depth_profile = next(p for p in depth_sensor.profiles if p.stream_type() == rs.stream.depth)
color_profile = next(p for p in color_sensor.profiles if p.stream_type() == rs.stream.color)
```
I think it enough abstract filter for profile because I not found a default profile in this API.
Please assists direction.